docs/FeatureManagement/FeatureFlagv2.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+        "items": {
+            "$id": "#/properties/variants/items",
+            "type": "object",
+            "title": "Variant",
+            "required": [
+                "name"
+            ],
+            "properties":{
+                "name": {
+                    "$id": "#/properties/variants/items/properties/name",
+                    "type": "string",
+                    "title": "Variant Name",
+                    "description": "The name used to refer to a feature variant.",
+                    "default": "",
+                    "pattern": "^(.*)$"
+                },
+                "configuration_value": {
+                    "$id": "#/properties/variants/items/properties/configuration_value",
+                    "type": ["string", "null", "number", "object", "array", "boolean"],
+                    "title": "Variant Configuration Value",
+                    "description": "The inline value set for this feature variant."
+                },
+                "configuration_reference": {
+                    "$id": "#/properties/variants/items/properties/configuration_reference",
+                    "type": "string",
+                    "title": "Variant Configuration Reference",
+                    "description": "The string used to reference a path in the configuration and set the variant's value equal to the value at that path.",
+                    "default": "",
+                    "pattern": "^[a-zA-Z0-9]+(:[a-zA-Z0-9]+)*$"
+                },
+                "status_override": {
+                    "$id": "#/properties/variants/items/properties/status_override",
+                    "type": "string",
+                    "title": "Variant Status Override",
+                    "description": "Overrides the enabled state of the feature according to this value when this variant is used. Does not override the state if value is None.",
+                    "enum": [
+                        "None",
+                        "Enabled",
+                        "Disabled"
+                    ],
+                    "default": "None"
+                }
+            }
+        }
+      },
+      "allocation": {
+        "$id": "#/properties/allocation",
+        "type": "object",
+        "title": "Variant Allocation",
+        "description": "Determines how variants should be allocated for the feature to various users.",
+        "required": [],
+        "properties": {
+            "default_when_disabled": {
+                "$id": "#/properties/allocation/properties/default_when_disabled",
+                "type": "string",
+                "title": "Variant Allocation Default When Disabled",
+                "description": "Specifies which variant should be used when a variant is requested while the feature is considered disabled.",
```

Comment:
Maybe we can simplify the sentence a little bit. Does this convey the same meaning?

Specifies which variant should be used when ~a variant is requested while~ the feature is considered disabled.